src/index.js

```diff
@@ -6,7 +6,7 @@ const execa = require('execa')
 const { YOUTUBE_DL_PATH } = require('./constants')
 
 const args = (url, flags = {}) =>
-  [].concat(url, dargs(flags, { useEquals: false })).filter(Boolean)
```

Comment:
hum can you test if 

```
[].concat(`"${url}"`, dargs(flags, { useEquals: false })).filter(Boolean)
```

works well? If yes I prefer that way since it will be more readable and easy to maintain

Reply:
Well, it works, but only with `{ defaultSearch: "ytsearch" }` which seems fine and you won't have to change anything.
This patch just felt more logical.